src/modules/Contribute/managers/ServiceManager.ts

```diff
@@ -24,60 +23,15 @@ export const addService = async ({
     accept: 'application/vnd.github.v3+json',
   };
 
-  const { origin } = new URL(url);
+  const prTitle = `Add ${name} - ${documentType}`;
+  const filePath = `declarations/${name}.json`;
 
-  const issueTitle = `Add ${name} - ${documentType}`;
-  const issueBodyCommon = `
-You can see the work done by the awesome contributor here:
-${url}
-
-Or on your local:
-${url.replace(origin, 'http://localhost:3000')}
-
-Or you can see the JSON generated here:
-\`\`\`json
-${JSON.stringify(json, null, 2)}
-\`\`\`
-
-You will need to create the following file in the root of the project: \`services/${name.trimEnd()}.json\`
-
-`;
-  const labelName = process.env.GITHUB_LABEL_ADD || 'add';
-
-  await createLabel({
+  return createPull({
     ...commonParams,
-    name: labelName,
-    description: 'Automatically created by Open Terms Archive Contribution Tool',
-    color: 'C2E0C6',
+    targetBranch: 'main',
+    newBranch: snakeCase(prTitle),
```

Comment:
Until now, the contribution tool did not take into consideration this ID.
2 solutions
- add a new field called ID in the form and leave the user write whatever he wants
- derive it from the name (It seems from the [doc](https://github.com/OpenTermsArchive/contrib-declarations/blob/main/CONTRIBUTING.md#service-id that it can be done)

Reply:
I created a function and added a test.
I did not yet take into account the change of cyrillic and chinese characters but it could be done pretty easily when we need it